packages/flutter/test/widgets/framework_test.dart

```diff
@@ -1694,6 +1694,26 @@ The findRenderObject() method was called for the following element:
 
     expect(inheritedElement.hashCode, identityHashCode(inheritedElement));
   });
+
+  testWidgets('doesDependOnInheritedElement', (WidgetTester tester) async {
+    final _TestInheritedElement ancestor = _TestInheritedElement(
+        const Directionality(
+            textDirection: TextDirection.ltr, child: Placeholder()));
```

Comment:
nit: nit formatting here should be:

```suggestion
    final _TestInheritedElement ancestor = _TestInheritedElement(
      const Directionality(
        textDirection: TextDirection.ltr, 
        child: Placeholder(),
      ),
    );
```

Reply:
This and the next nit are still unaddressed.

Reply:
done